shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/datasource/config/PoolConfiguration.java

```diff
@@ -40,6 +40,8 @@ public final class PoolConfiguration {
     private final Integer minPoolSize;
     
     private final Boolean readOnly;
-    
+
+    private final Boolean registerMbeans;
```

Comment:
Could we set this by `customProperties`? `registerMbeans` is Hikari's property, but PoolConfiguration is a common configuration.